Sources/Sentry/Public/SentryDefines.h

```diff
@@ -142,3 +142,84 @@ typedef NS_ENUM(NSInteger, SentryTransactionNameSource) {
     kSentryTransactionNameSourceComponent,
     kSentryTransactionNameSourceTask
 };
+
+/**
+ * A time duration.
+ */
+typedef NS_ENUM(NSInteger, SentryDurationUnit) {
```

Comment:
I will throw these enums away if we agree that SentryMeasurement and SentryUnit is the way to go.